src/main/java/net/sf/jabref/gui/FindUnlinkedFilesDialog.java

```diff
@@ -532,10 +533,12 @@ private void startSearch() {
                         @Override
                         public void stateChanged(ChangeEvent e) {
                             counter++;
-                            progressBarSearching.setString(counter + " files found");
+                            SwingUtilities.invokeLater(() -> progressBarSearching
```

Comment:
This is hard to read. Maybe determine the localized string before calling `SwingUtil.invokeLater()`